.github/workflows/npmpublish.yml

```diff
@@ -0,0 +1,30 @@
+# This workflow will run tests using node and then publish a package to GitHub Packages when a release is created
+# For more information see: https://help.github.com/actions/language-and-framework-guides/publishing-nodejs-packages
+
+name: NPM Publish
+
+on:
+  release:
+    types: [published]
```

Comment:
@amphro There were several options around this but published seemed to make sense for at least this first iteration.